src/components/FormNavigation/FormNavigation.tsx

```diff
@@ -24,29 +27,49 @@ const FormNavigation: React.FC<FormNavigationProps> = ({
   onFormSave,
   previousPage,
   onDestroySession,
-}) => (
-  <ButtonGroup className="openforms-form-navigation" direction="column">
-    {submitButton}
-
-    {previousPage && <PreviousLink to={previousPage} onClick={onNavigatePrevPage} position="end" />}
-
-    {onFormSave && (
-      <LinkButton
-        name="save"
-        onClick={onFormSave}
-        className="openforms-form-navigation__save-button"
-      >
-        <Icon>
-          <FAIcon icon="" className="fa-fw" />
-        </Icon>
-        <Literal name="saveText" />
-      </LinkButton>
-    )}
-
-    {!hideAbortButton && (
-      <AbortButton isAuthenticated={isAuthenticated} onDestroySession={onDestroySession} />
-    )}
-  </ButtonGroup>
-);
+}) => {
+  const {backToTopText, backToTopRef} = useContext(ConfigContext);
+  const backToTopNode = backToTopRef ? document.getElementById(backToTopRef) : null;
+
+  return (
+    <ButtonGroup className="openforms-form-navigation" direction="column">
+      {submitButton}
+
+      {previousPage && (
+        <PreviousLink to={previousPage} onClick={onNavigatePrevPage} position="end" />
+      )}
+
+      {onFormSave && (
+        <LinkButton
+          name="save"
+          onClick={onFormSave}
+          className="openforms-form-navigation__save-button"
+        >
+          <Icon>
+            <FAIcon icon="" className="fa-fw" />
+          </Icon>
+          <Literal name="saveText" />
+        </LinkButton>
+      )}
+
+      {!hideAbortButton && (
+        <AbortButton isAuthenticated={isAuthenticated} onDestroySession={onDestroySession} />
+      )}
+
+      {!!backToTopText && backToTopNode && (
+        <LinkButton
+          name="backToTop"
+          className="openforms-form-navigation__backtotop-button"
+          onClick={() => backToTopNode.scrollIntoView({behavior: 'smooth'})}
```

Comment:
is this because otherwise with hash-based routing things would break? Or is there another reason this can't be a plain `<a href="#ref">...</a>` construct with native browser link-click behaviour?